src/android/app/src/main/java/io/github/mandarine3ds/mandarine/fragments/SetupFragment.kt

```diff
@@ -370,6 +401,84 @@ class SetupFragment : Fragment() {
     private lateinit var microphoneCallback: SetupCallback
     private lateinit var cameraCallback: SetupCallback
 
+    private fun showStaticThemeSelectionDialog(onComplete: () -> Unit) {
+        val preferences = PreferenceManager.getDefaultSharedPreferences(requireContext())
+        val themeColors = resources.getStringArray(R.array.staticThemeNames)
+        val currentThemeColor = preferences.getInt(Settings.PREF_STATIC_THEME_COLOR, 0)
+
+        MaterialAlertDialogBuilder(requireContext())
+            .setTitle(R.string.set_up_theme_settings)
+            .setSingleChoiceItems(themeColors, currentThemeColor) { _, which ->
+                preferences.edit().putInt(Settings.PREF_STATIC_THEME_COLOR, which).apply()
+            }
+            .setPositiveButton(android.R.string.ok) { _, _ ->
+                onComplete()
+            }
+            .setNegativeButton(android.R.string.cancel, null)
+            .show()
+    }
+
+    private fun showMaterialYouAndBlackThemeDialog() {
+        val preferences = PreferenceManager.getDefaultSharedPreferences(requireContext())
+        val switchContainer = LinearLayout(requireContext()).apply {
+            orientation = LinearLayout.VERTICAL
+            setPadding(64, 16, 64, 32)
```

Comment:
Looks a bit "hacky", but seems to be needed to work, so i'll let it go like this for now.